Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPIBookmarks.mm

```diff
@@ -134,6 +134,23 @@ @interface TestBookmarksDelegate : NSObject <WKWebExtensionControllerDelegatePri
     return nil;
 }
 
+static NSMutableDictionary *findBookmarkAndParentArrayInMockTree(NSMutableArray *rootChildren, NSString *bookmarkId)
```

Comment:
Nit: I think `rootChildren` could just be `children` - since due to the recursion it won't always be the root.